src/assets/styles/50-utilities/_scaffolding.scss

```diff
@@ -36,5 +36,31 @@
 
   @include mq(md) {
     @include grid-column(1, 13);
+    position: relative;
+  }
+}
+
+
+// "wide" content has to appear above sticky content
+// such as the in-page nav
+.u-scaffold-grid-item--wide > * {
+  @include mq(md) {
+    position: relative;
+    z-index: var(--z-medium);
+  }
+}
+
+// linear gradient applied to give a fading out appearance to sticky content layered behind it
+.u-scaffold-grid-item--wide:after {
+  @include mq(md) {
+    background: linear-gradient(180deg, var(--hack-safari-transparent-white) 0, var(--colour-white) 10%, var(--colour-white) 90%, var(--hack-safari-transparent-white) 100%);
+    bottom: -100px;
+    content: '';
+    left: 0;
+    pointer-events: none;
+    position: absolute;
+    right: 0;
+    top: -100px;
+    z-index: var(--z-low);
   }
 }
```

Comment:
I really like this effect and how it's applied... just a thought, would it be clearer if applied via it's own utility selector? `.u-sticky-fade` or something to that effect?

Reply:
Alternatively, a mixin may be appropriate, too. I think this would give it better clarity of purpose & reusability

Reply:
Mixin used in preference, see comments below